test/DivaBrowser.test.js

```diff
@@ -41,6 +41,45 @@ describe.each(['firefox', 'chrome', 'safari'])('Tests on %s', (title) => {
     });
   });
 
+  describe('Boundingbox selecting', () => {
+    test('Select syl bbox', async () => {
+      await browser.wait(until.elementLocated(By.id('displayBBox')), 10000);
+      let displayButton = await browser.findElement(By.id('displayBBox'));
+      if (title !== 'safari') {
```

Comment:
Is it absolutely necessary to have different actions for different browsers?

Reply:
safari was not cooperating :( I added a comment before the first if/else